ui/app/AppLayouts/Chat/components/ProfilePopup.qml

```diff
@@ -371,15 +387,20 @@ ModalPopup {
 
         StyledButton {
             anchors.right: parent.right
-            anchors.rightMargin: addToContactsButton.width + 32
+            anchors.rightMargin: isBlocked ? 0 : addToContactsButton.width + 32
             btnColor: Style.current.lightRed
             btnBorderWidth: 1
             btnBorderColor: Style.current.grey
             textColor: Style.current.red
-            //% "Block User"
-            label: qsTrId("block-user")
+            label: isBlocked ? qsTr("Unblock User") : qsTr("Block User")
```

Comment:
Could you put the `qsTr`s on two different lines for the context comments can be put above them correctly

Reply:
Done